ush/python/pygw/src/pygw/schema.py

```diff
@@ -0,0 +1,891 @@
+"""schema is a library for validating Python data structures, such as those
+obtained from config-files, forms, external services or command-line
+parsing, converted from JSON/YAML (or something else) to Python data-types."""
+
+import inspect
+import re
+
+# The following are specific to global-workflow.
+import logging
+from typing import Dict
+from pydoc import locate
+from pygw import yaml_file
+from pygw.attrdict import AttrDict
+from pygw.logger import Logger, logit
```

Comment:
We should try to keep things as lean and close to the upstream package as possible.
```suggestion
```
I don't think we need to add these.
See comments below.